src/inspector/view/qml/MuseScore/Inspector/text/TextInspectorView.qml

```diff
@@ -127,23 +127,13 @@ InspectorSectionView {
                 navigationRowStart: styleSection.navigationRowEnd + 1
 
                 titleText: qsTrc("inspector", "Size")
+                measureUnitsSymbol: qsTrc("inspector", "pt")
                 propertyItem: root.model ? root.model.fontSize : null
 
-                model: [
-                    { text: "8",  value: 8 },
-                    { text: "9",  value: 9 },
-                    { text: "10", value: 10 },
-                    { text: "11", value: 11 },
-                    { text: "12", value: 12 },
-                    { text: "14", value: 14 },
-                    { text: "16", value: 16 },
-                    { text: "18", value: 18 },
-                    { text: "22", value: 22 },
-                    { text: "24", value: 24 },
-                    { text: "30", value: 30 },
-                    { text: "36", value: 36 },
-                    { text: "48", value: 48 }
-                ]
+                decimals: 2
```

Comment:
Wouldn't one decimal be enough? (@bkunda?)

Reply:
Oh yes, one decimal is definitely enough for text in this context.

Reply:
fixed